OpenSim/Simulation/Model/ModelVisualizer.cpp

```diff
@@ -245,7 +245,7 @@ bool ModelVisualizer::
 findGeometryFile(const Model& aModel, 
                  const std::string&          geoFile,
                  bool&                       geoFileIsAbsolute,
-                 SimTK::Array_<std::string>& attempts) const
+                 SimTK::Array_<std::string>& attempts)
```

Comment:
Logically, this method is const. The other option is to const_cast `attempts` whenever you use it. But I think this is fine for now. No change neessary.